R/model-design-tools.R

```diff
@@ -191,8 +193,8 @@ enw_effects_metadata <- function(design) {
 #' effects <- enw_effects_metadata(design)
 #' enw_add_pooling_effect(effects, prefix = "b")
 enw_add_pooling_effect <- function(effects, var_name = "sd",
-                                   finder_fn = startsWith, ...) {
-  effects <- data.table::setDT(effects)
+                                   finder_fn = startsWith, ..., copy = TRUE) {
```

Comment:
I think we can avoid passing through here as can't see the user desire to have a copy?

Reply:
you mean, always assume copy = FALSE? that seems consistent with how it seems to be used internally

Reply:
yes exactly

Reply:
righto, making it always copy = FALSE. side note: from a naming-conventions standpoint, perhaps have `..._add_...` methods always modify in place?